package.json

```diff
@@ -4,7 +4,7 @@
     "description": "Maintenance Tracker App is an application that provides users with the ability to reach out to operations or repairs department regarding repair or maintenance requests and monitor the status of their request.",
     "main": "app.js",
     "scripts": {
-        "test": "echo \"Error: no test specified\" && exit 1",
+        "test": "mocha --exit --require babel-register ./Server/tests/**/*.test.js",
         "start": "nodemon --exec babel-node app.js"
```

Comment:
Do not use nodemon in the start script but you can use it in the start:dev script